src/main/java/com/iabtcf/encoder/TCModelEncoder.java

```diff
@@ -0,0 +1,16 @@
+package com.iabtcf.encoder;
+
+import com.iabtcf.GDPRTransparencyAndConsent;
+
+/**
+ * @author ewhite 2/25/20
+ */
+public class TCModelEncoder {
+
+	public static String encode(final GDPRTransparencyAndConsent tc) {
+		// TODO - actually encode to a web safe 64 encoded string
+		//  (will need to modify class access/package structure to give access to the builders of objects)
+		//  Be sure to add tests!
+		return "";
+	}
+}
```

Comment:
lets park this for now

Reply:
I agree. Probably better to not have anything there while it doesn't work. Will remove.